put_all_brand.py

```diff
@@ -0,0 +1,17 @@
+import requests
+import pytest
+
+url = 'https://automationexercise.com/api/brandsList'
+
+def put_all_brand():
+    updated_data = {
+        'brandName': 'India cement',
+        'brandStatus': 'Active'
+    }
+    response = requests.put(url, json=updated_data)
+
+    # assert the status code is 200 or 204
+    assert response.status_code in [200, 204]
```

Comment:
please validate the content

Reply:
Made the required changes kindly review it.